#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_linkcolor.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Run htmldoc with --linkcolor #0000ff and --no-embedfonts

@DPATCH@
diff -urNad privoxy~/utils/ldp_print/ldp_print privoxy/utils/ldp_print/ldp_print
--- privoxy~/utils/ldp_print/ldp_print
+++ privoxy/utils/ldp_print/ldp_print
@@ -54,7 +54,7 @@
 
 my($cmd) = "htmldoc --size universal --bodyfont helvetica --fontsize 8 " .
            "-t pdf -f ${fname_wo_ext}.pdf --firstpage p1 --titlefile title.html" .
-           " body.html --footer c.1";
+           " body.html --footer c.1 --linkcolor #0000ff --no-embedfonts";
 
 # For postscript output; append onto the above cmd string:
 #
